velox/exec/OrderBy.cpp

```diff
@@ -202,6 +202,19 @@ void OrderBy::ensureInputFits(const RowVectorPtr& input) {
           0, outOfLineBytes - (rowsToSpill * outOfLineBytesPerRow)));
 }
 
+void OrderBy::reclaim(uint64_t targetBytes) {
+  VELOX_CHECK(canReclaim());
+  VELOX_CHECK(!operatorCtx_->driver()->state().isOnThread());
```

Comment:
Does this line mean `::reclaim` cannot be called from the same task thread?

Reply:
It means we won't reclaimer from a running driver. We will first pause a task before reclaim memory from its operators.